Resources/config/services.yml

```diff
@@ -94,6 +96,10 @@ services:
     public: true
     lazy: true
 
+  lch.media.behavior.reflection.class_analyzer:
+    class: '%lch.media.behavior.reflection.class_analyzer.class%'
+    public: false
+
```

Comment:
Same remark as above, this is not a behavior. You can rename it after moving it elsewhere

Reply:
Ok, i will change it